pkg/services/accesscontrol/resourcepermissions/store.go

```diff
@@ -703,3 +742,90 @@ func managedPermission(action, resource string, resourceID, resourceAttribute st
 		Scope:  accesscontrol.Scope(resource, resourceAttribute, resourceID),
 	}
 }
+
+/*
+ACTION SETS
+
+Stores actionsets IN MEMORY
+*/
+// - To grant access to users/teams/basic roles during runtime, users will go through the managed permissions view and access-control will be enforced based on their permissions.
```

Comment:
I'm wondering if we need a whole interface for this, or if we can start with simpler code and only introduce it when needed.

Reply:
yea, i'd thought id do the interface to make testing easier to implement. but il remove it. it's a bit overkill i guess